backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/navigationbar/internal/BpkTopNavBarImpl.kt

```diff
@@ -64,7 +65,7 @@ internal fun BpkTopNavBarImpl(
     contentColor = BpkTheme.colors.textPrimary,
     elevation = animateDpAsState(targetValue = if (fraction == 0f) BpkDimension.Elevation.Sm else 0.dp).value,
     shape = RectangleShape,
-    modifier = modifier,
+    modifier = modifier.zIndex(Float.MAX_VALUE),
```

Comment:
only concern with using `max` here is there might be cases where we DO want to show something on top of a toolbar (can think of a dropdown menu for example from the nav bar unless they're drawn in a separate window)

Reply:
Let's use 1 for now - we can always make it configurable/change in case it's needed